src/Xrefcheck/Verify.hs

```diff
@@ -412,6 +455,41 @@ verifyRepo
       ExternalLoc -> CacheUnderKey rLink
       _           -> NoCaching
 
+checkCopyPaste :: FilePath -> [Reference] -> [CopyPasteCheckResult]
+checkCopyPaste file refs = do
+  let getLinkAndAnchor x = (rLink x, rAnchor x)
+      groupedRefs =
+          L.groupBy ((==) `on` getLinkAndAnchor) $
```

Comment:
Due to a recent feature (#211) depending on the configured Markdown flavour, anchors are case-insensitive or not. Maybe this `==` should behave in that same way? And be case-insensitive for `rAnchor x` depending on the Markdown flavour.

Also, I think that if we had a custom datatype for anchors that, depending on the Markdown flavour, had a case-insensitive `Eq` implementation, this would be automatically preserved for new features. How do you see to create an issue with that refactor @Martoon-00?

Reply:
Well, here we check for the copypaste, and if someone copypastes an anchor, its case preserves. If the case is different, it's almost always done intentionally. So I think here we can just use existing anchors.

Reply:
> If the case is different, it's almost always done intentionally.

That's a pretty fair observation.

> had a case-insensitive Eq implementation, this would be automatically preserved for new features. How do you see to create an issue with that refactor @Martoon-00?

So this turns to be not related to this PR, but I agree. @aeqz Please create a ticket for this.

Reply:
For the record: #255 has been created.